install_cluster/defaults/main.yml

```diff
@@ -4,12 +4,14 @@
 install: True
 secure: False
 debug: False
-
+download: False
+sno: True
```

Comment:
Try to make defaults the same where you can as it makes it easier to move code between roles.